src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/CustomConverterTests/CustomConverterTests.Object.cs

```diff
@@ -732,5 +734,34 @@ static void Verify(JsonSerializerOptions options)
             options.Converters.Add(new SystemObjectNewtonsoftCompatibleConverter());
             Verify(options);
         }
+
+        [Fact]
+        public static void CanCustomizeSystemObjectSerialization()
+        {
+            var options = new JsonSerializerOptions { Converters = { new CustomSystemObjectConverter() } };
+
+            string expectedJson = "\"Object\"";
+            string actualJson = JsonSerializer.Serialize(new object(), options);
+            Assert.Equal(expectedJson, actualJson);
+        }
+
+        private class CustomSystemObjectConverter : JsonConverter<object>
+        {
+            public override object? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options) => throw new NotImplementedException();
+            public override void Write(Utf8JsonWriter writer, object value, JsonSerializerOptions options)
+            {
+                if (value?.GetType() == typeof(object))
+                {
+                    // check that serialization for values of
+                    // runtime type object can be customized.
+                    writer.WriteStringValue("Object");
+                }
+                else
+                {
+                    throw new NotSupportedException();
```

Comment:
Do we have a test where the runtime type is something else which means the converter would not be called?

Reply:
The converter would still get called for any runtime type. I think we already have tests in that same file that check for that behavior.

Reply:
I have simplified the test to provide more clarity on the nature of the behavioral change.